test/e2e/tests_definitions.go

```diff
@@ -1240,6 +1240,7 @@ const (
 type Scenario interface {
 	SetInfra(infrastructure Infra)
 	SetVersions(versions ...string)
+	SetInitKubeOneVersion(version string)
```

Comment:
Given that only 1 scenario needs it, it looks like this is the case for [interface smuggling](https://utcc.utoronto.ca/~cks/space/blog/programming/GoInterfaceSmuggling).

Reply:
I.e. make SetInitKubeOneVersion() own interface, and use type assertion if scenario implements it.

Reply:
@kron4eg Good point. Fixed, PTAL.